site/docs/compositemark.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+menu: docs
+title: Composite Mark
+permalink: /docs/compositemark.html
+---
+
+Composite marks are "macros" for more complex layered mark specs. Layered mark specs create visualizations with a series of basic mark types.
+
+| Property      | Type          | Description    |
+| :------------ |:-------------:| :------------- |
+| `mark`      | String        | The CompositeMark type. Supported composite marks include: "box-plot". |
+
+## Box Plot
+
+`box-plot` composite mark represents a box-plot or box and whisker plot. The lower and upper whiskers extend to the min and max respectively. The middle tick in the box represents the median. The lower and upper part of the box represents quartile 1 and 3 respectively.
+
+`box-plot` supports the following properties: color, size, opacity.
+
+### Vertical 1D Boxplot
```

Comment:
- Briefly describe that `aggregate` of the continuous field is implicitly `box-plot`.